internal/utils/csi.go

```diff
@@ -393,3 +392,11 @@ func FuseMountOptionsContainerArg(options map[string]string) string {
 		"",
 	)
 }
+
+func TopologyDomainLaels(options []string) string {
```

Comment:
1. a misspelling of the word "labels"
2. We should align on the naming convention for dynamic container args (`<arg name>+ContainerArg`) 
```suggestion
func DomainLabelsContainerArg(options []string) string {
```